docs/content/users/install/shell-completion.md

```diff
@@ -37,6 +37,7 @@ Shells like bash and zsh need help to do this though, they have to know what the
         tar -zxf ddev_shell_completion_scripts.${VERSION}.tar.gz
         ```
     2. Run `sudo mkdir -p /usr/share/bash-completion/completions && sudo cp ddev_bash_completion.sh /usr/share/bash-completion/completions/ddev`. This deploys the `ddev_bash_completion.sh` script where it needs to be. Again, every Linux distro has a different technique, and you may have to figure yours out.
+    3. Remember to reload your shell with `$ exec zsh` or by closing and reopening your terminal.
```

Comment:
Yeah, just sourcing should do the job.

Reply:
It didn't work for me until I closed the terminal (in one case), and in the other case, until I ran `exec zsh`.

Reply:
You may be missing the required autoload in your zsh setup. However, I use bash and am certainly no expert on zsh, especially Oh-My-Zsh. Zsh can be set up so many different ways it can fry the brain. I think one of those ways bit you here.

Reply:
> Yeah, just sourcing should do the job.

I had to run:

```
autoload -Uz compinstall && compinstall
```
And then reload my shell before completions worked. 

Mac OS Ventura 13.5 running zsh. Installed ddev via homebrew.

```
zsh --version
zsh 5.9 (x86_64-apple-darwin22.0)
```